src/main/protobuf/v1/token.proto

```diff
@@ -10,7 +10,7 @@ message TokenV1 {
   required string id = 1;
   repeated PartyRelationshipBindingV1 legals = 2;
   required string validity = 3;
-  required string seed = 4;
+  required string applicationId = 4;
```

Comment:
what is the `applicationId`?

Reply:
See [comment](https://github.com/pagopa/pdnd-interop-uservice-party-management/pull/36#discussion_r761249866).